.travis.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+##
+#  Copyright (c) DataStax, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+##
+
+#Debug statements [if needed]
+#set -x #Trace
+#set -n #Check Syntax
+
+WORKING_DIRECTORY="$(pwd)"
```

Comment:
These two variables not used?

Reply:
Yeah sorry ... those are my boiler plate variables I use in bash scripts; will remove.